code/stable/gamephysics/SRS/HTML/GamePhysics_SRS.html

```diff
@@ -904,7 +904,7 @@ <h3>Theoretical Models</h3>
                     </tr>
                     <tr>
                       <th>Equation</th>
-                      <td>\[{\symbf{F}_{1}}=-{\symbf{F}_{2}}\]</td>
+                      <td>\[\symbf{F}_{1}=-\symbf{F}_{2}\]</td>
```

Comment:
Here the change seems like an improvement, but I also don't know why it happened! Why were there extra {} before? Sometimes they were needed to make some things behave better. But I have no idea why your changes would have cause this too.

Reply:
The extraneous `P.Row`s were the culprit for that. My changes try to use them as frequently as possible.